src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -17,7 +17,9 @@ commands:
   - command: helm repo update
   - command: helm install --set logLevel=trace --set image.tag=dev
       --set image.repository=localhost/redpanda-operator --namespace redpanda --create-namespace
-      redpanda-operator redpanda/operator
+      redpanda-operator redpanda/operator --set additionalCmdFlags="{--additional-controllers=all}"
+  - command: "sh -c 'until kustomize build config/e2e-v2-tests 2>> tests/_e2e_artifacts/kustomize-output.txt | kubectl apply -f - 1>> tests/_e2e_artifacts/kubectl-output.txt 2>> tests/_e2e_artifacts/kubectl-error-output.txt; do sleep 0.5; done'"
```

Comment:
I don't like this change as we should relay only on official helm chart. Any change that needs to update helm chart should be guarded/behind feature flag (you already did this `additional-controllers=all`), so that order should be:

1. Redpanda PR with feature behind feature flag
2. Helm chart with all changes required, because new feature requires it
3. Enable feature by default in operator E2E test

Reply:
Will remove this soon